tools/lint/lint.py

```diff
@@ -1025,6 +1025,11 @@ def lint(repo_root, paths, output_format, ignore_glob=None, github_checks_output
 
     if jobs == 0:
         jobs = multiprocessing.cpu_count()
+    if sys.platform == 'win32':
```

Comment:
Should this be either just inside the `if jobs == 0` block, or at least log a warning if it's going to clamp to a number lower than the explictly user-provided number?

Reply:
Good point. I agree that if a user passes a job count then they are responsible for what happens. I moved the check inside the conditional.